components/02-components/secondary-navigation/01-side-navigation/README.md

```diff
@@ -0,0 +1,17 @@
+A hierarchical vertical navigation on the left side of a page that helps users navigate between pages within a section of the website.
```

Comment:
I think it would be helpful to expand on navigation -- that the links on the side nav actually click out to new pages and doesn't do a non-refreshed (ajax) load and not to be confused with the other panels that jump to sections on content on the same page.